app/validators/contact_data_validator.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class ContactDataValidator < ActiveModel::EachValidator
+  # Contact Type regexps
+  PHONE_REGEX = /\A(\+?\d)?[\d]+\z/i.freeze
```

Comment:
Moved Regexps to Validator
It is very likely that we will be adding new validations to "data" of Contact, so it's best to create Validator now